README.md

```diff
@@ -800,6 +800,13 @@ console.log('three');
   two
   ```
 
+### AngularJS Questions:
+
+* What is AngularJS?
+  AngularJS is a structural framework for dynamic web apps. It lets you use HTML as your template language and lets you extend HTML's syntax to express your application's components clearly and succinctly. AngularJS's data binding and dependency injection eliminate much of the code you would otherwise have to write. And it all happens within the browser, making it an ideal partner with any server technology.
```

Comment:
no i to trochę jest copy paste ze strony angulara, może tak bardziej po ludzku i bez zachwalania? ;)